Sezai/index.js

```diff
@@ -0,0 +1,121 @@
+//These exercises are aimed at making you more comfortable with using conditionals as well as operators. Use "if" statements to complete the following exercises. Print your results to the console.
+
+//Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range.
+
+
+let num1 = 49;
+let num2 = 100;
+
+if(num1 >=50 && num2 <= 90){
+    console.log("they are in the range of 50 and 90 ==> ");
+}else {
+    console.log("they are NOT in the range of 50 and 90");
+}
+
+
+//Check if three given integer values are in the range 50 to 99 (inclusive). Print true if one or more of them are in the range.
+
+let num3 = 100;
+
+if(num1>=50 || num2 <=99 || num3 <=99){
+    console.log("they are in the range of 50 and 99");
```

Comment:
Here also you have to check for every variable if they are in the range separately and then use ||